examples/lighting-app/nrf5/BUILD.gn

```diff
@@ -51,15 +51,21 @@ nrf5_sdk("sdk") {
 
   defines += [
     "USE_APP_CONFIG",
+    "ENABLE_CHIP_SHELL",
+    "NRF_SHELL_STREAMER",
     "MBEDTLS_PK_WRITE_C",
     "MBEDTLS_X509_CREATE_C",
     "MBEDTLS_X509_CSR_WRITE_C",
+    "OPENTHREAD_CONFIG_CLI_TRANSPORT=OT_CLI_TRANSPORT_CONSOLE",
```

Comment:
Could these be added to the OpenThread config file?

Reply:
Since we still have otcli on other platforms, I prefer to keep this line and remove it after we replace the native otcli by the otcli in chip_shell on other platforms.